### PR TITLE
Explain why the button is disabled

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Comment by jSylvestre** [comment]
image

File: Anlab.Mvc/Client/components/OrderForm.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
         this.state = { ...initialState };
     }
     validate = () => {
-        const valid = this.state.quantity > 0 && !!this.state.project.trim();
+        const valid = this.state.quantity > 0 && this.state.quantity <= 100 && !!this.state.project.trim();
```
Comment (srkirkland) [inline Anlab.Mvc/Client/components/OrderForm.tsx]: @jSylvestre is there a way we can just ask the quantity box if it is valid, or have a this.state.quantityValid property?  That way we aren't tied in multiple places to this range limit

Comment (jSylvestre) [inline Anlab.Mvc/Client/components/OrderForm.tsx]: @srkirkland I was asking @jpknoll about this. There are some frameworks we could import, but may not be worth it unless the form gets more complex.

Comment (srkirkland) [inline Anlab.Mvc/Client/components/OrderForm.tsx]: @jSylvestre Yea not too interested in adding a forms lib, was just hoping there was a way of not duplicating the logic

**Pull request merged**